packages/sdk/src/ClientSide.ts

```diff
@@ -22,10 +25,10 @@ export class ClientSide {
 			query.code = params
 		} else {
 			query.code = params.code
-			query.item = params.items
+			query.item = params.order?.items
 			query.amount = params.amount
 			query.metadata = params.metadata
-			query.order = { metadata: params.orderMetadata }
+			query.order = { metadata: params.order?.metadata }
```

Comment:
If you're making this change to the input `params`, then you'd need to update all examples as well:

https://github.com/voucherifyio/voucherify-js-sdk/blob/95304a121c5f1e3a7889054283f4a7f5b7912f9d/examples/with-nodejs-typescript/client-validate-voucher.ts#L12-L19

Is this necessary? I mean, probably for 1st public release, it'd be good to keep the input `params` as users are used to in old JS SDK?

Reply:
I'm not so sure, the 'orderMetadata' key sicks out for me - it's camel cased, not snake cased + its unique in scope of the whole project (/[a-zA-Z]Metadata/).

If we're going with TS and documenting this whole process, i would keep those params objects as similiar as possible, but it's just my opinion.